modules/contentbox/models/content/BaseContent.cfc

```diff
@@ -1236,7 +1236,7 @@ component 	persistent="true"
 	* Build content cache keys according to sent content object
 	*/
 	string function buildContentCacheKey(){
-		return "cb-content-#cgi.http_host#-#getContentType()#-#getContentID()#-#i18n.getfwLocale()#";
+		return "cb-content-#cgi.http_host#-#cgi.query_string#-#getContentType()#-#getContentID()#-#i18n.getfwLocale()#";
```

Comment:
We have encountered this issue with our photo gallery module and our data listing widget when trying to using paging.  Up to this point we have had to turn of page caching to be able to use caching because the caching hash does not include the query string.  I feel that this is a valid change because without it, you do not have the option of having url derived content after the first request. 

One example of this caching scheme being used elsewhere is in the FullCalendar JS library.  [https://fullcalendar.io/docs/event_data/events_json_feed/#caching](Here) they show how they use a unique identifier in the url to prevent caching when using a JSON feed.